modules/CmpdReg/src/client/src/AddAlias.js

```diff
@@ -104,7 +104,7 @@
     AddAliasController.prototype.initialize = function() {
       this.collection = this.options.collection;
       this.listOfAliases = new AliasListList();
-      this.listOfAliases.type = "parentaliaskinds";
+      this.listOfAliases.type = "aliases/parentAliasKinds";
```

Comment:
These were already setup in the roo /api/v1 routes via `aliases/parentAliasKinds`. Kind of awkward but type = the url this controller calls.